fdbclient/VersionVector.h

```diff
@@ -64,6 +69,51 @@ struct VersionVector {
 		return iter->second;
 	}
 
+	void clear() {
+		versions.clear();
+		maxVersion = invalidVersion;
+	}
+
```

Comment:
- documentation here would be good for readers unfamiliar with this strategy.
- later, we may want to have a way to select how the vector is encoded. `knob={none, strategy1,  strategy2,}` etc).

Reply:
Done.